privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/ui/PrivacyDashboardHybridViewModel.kt

```diff
@@ -182,16 +193,16 @@ class PrivacyDashboardHybridViewModel @Inject constructor(
 
         viewModelScope.launch(dispatcher.io()) {
             if (enabled) {
-                userWhitelistDao.delete(currentViewState().siteProtectionsViewState.site.domain)
+                userWhitelistDao.delete(currentViewState().siteViewState.domain)
```

Comment:
nit: we should ideally try to avoid using daos directly and instead start adding methods to UserAllowListRepository when needed. Not really asking to do it here but FYI :)

Reply:
Thanks, I've created a task to migrate all daos. https://app.asana.com/0/1202552961248957/1203198577387937/f

I will keep it under my radar.